vector/src/main/res/layout/adapter_item_vector_people_header.xml

```diff
@@ -49,6 +49,7 @@
         <ImageView
             android:id="@+id/heading_image"
             android:layout_width="20dp"
+            android:tint="?attr/vctr_general_icon_tint_color"

```

Comment:
Using `?vctr_list_header_primary_text_color` is sufficient, please do not create a new `vctr` attribute.

Reply:
Also, you may be interested in changing the tools:src attribute with `tools:src="@drawable/ic_material_expand_more_black"` in the 3 concerned layouts, for a better preview